docs/tutorials/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,535 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pylab.rcParams[\"figure.figsize\"] = (12, 8)\n",
+    "for i, optimizer in enumerate(optimizers):\n",
+    "    pylab.plot(\n",
+    "        converge_counts[i],\n",
+    "        abs(ref_value - converge_vals[i]),\n",
+    "        label=type(optimizer).__name__,\n",
+    "    )\n",
+    "pylab.xlabel(\"Eval count\")\n",
+    "pylab.ylabel(\"Energy difference from solution reference value\")\n",
+    "pylab.title(\"Energy convergence for various optimizers\")\n",
+    "pylab.yscale(\"log\")\n",
+    "pylab.legend(loc=\"upper right\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Gradients\n",
+    "\n",
+    "In Qiskit's variational algorithms, if the provided optimizer uses a gradient-based technique, the default gradient method will be finite differences. However, these classes include an option to pass custom gradients via the `gradient` parameter, which can be any of the provided methods within Qiskit's [gradient](https://qiskit.org/documentation/stubs/qiskit.algorithms.gradients.html) framework, which fully supports the use of primitives. This section shows how to use custom gradients in the VQE workflow.\n",
```

Comment:
```suggestion
    "In the variational algorithms, if the provided optimizer uses a gradient-based technique, the default gradient method will be finite differences. However, these classes include an option to pass custom gradients via the `gradient` parameter, which can be any of the provided methods within the [gradient](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.gradients.html) framework, which fully supports the use of primitives. This section shows how to use custom gradients in the VQE workflow.\n",
```
A couple of "Qiskit's" - I think the change to just "the" in both cases is ok - in the 2nd there is the link to gradients  - which needed to be updated too.

Maybe a quick last search for "qiskit.algorithms" anywhere in these files just to double check...

Reply:
I think that my search and replace doesn't do well with links, (I think that Pycharm renders markdown cells and this might hide them from the search) because searching for "qiskit.algorithms" was the first thing I did... ok, doing a second round